src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -133,6 +134,26 @@ internal static SafeSharedX509NameStackHandle SslGetClientCAList(SafeSslHandle s
             return handle;
         }
 
+        internal static bool AddExtraChainCertificates(SafeSslHandle sslContext, X509Chain chain)
+        {
+            if (chain != null)
```

Comment:
It looks like both callers check for null.  In what circumstance would chain be null here?  If it's not possible with the current calls, I'd suggest changing this to be a Debug.Assert.